hogql_parser/parser.cpp

```diff
@@ -1208,6 +1212,42 @@ class HogQLParseTreeConverter : public HogQLParserBaseVisitor {
     return text;
   }
 
+  VISIT(TagAttribute) {
+    string name = visitAsString(ctx->identifier());
+
+    if (ctx->columnExpr()) {
```

Comment:
To avoid calling `ctx->foo()` twice, there's the `auto foo_ctx = ctx->foo()` pattern (e.g. `auto column_expr_ctx = ctx->columnExpr()`)